mkosi/__init__.py

```diff
@@ -4236,13 +4244,32 @@ def run_verb(args: Args, images: Sequence[Config], *, resources: Path) -> None:
 
     check_workspace_directory(last)
 
+    if last.incremental == Incremental.strict:
+        if args.force > 1:
+            die("Cannot remove incremental caches when building with Incremental=strict")
```

Comment:
The usability here seems a bit weird, since if `Incremental=strict` is set in a config file, `mkosi -ff` doesn't work anymore without editing the file or setting `-i no`. Would doing
```suggestion
        if 1 < args.force < 3:
            die("Cannot remove incremental caches when building with Incremental=strict")
```
make sense here? Maybe something like `hint="Use --incremental=no"` would be enough.

Reply:
I think a hint is the way to go